src/InputStepper/index.js

```diff
@@ -118,24 +151,28 @@ class InputStepper extends React.Component<Props & ForwardedRef, State> {
           tabIndex={tabIndex}
           ref={forwardedRef}
           prefix={
-            <ButtonLink
-              disabled={disabled || value <= +minValue}
-              iconLeft={<MinusCircle color="secondary" />}
-              size={size}
-              onClick={this.decrementCounter}
-              transparent
-              component={PrefixSuffix}
-            />
+            <StyledButtonWrapper role="button" tabIndex="0" onKeyDown={this.handleKeyDecrement}>
```

Comment:
Shouldn't be this handled by `ButtonLink` itself? I mean all properties (can be added, make sense for me) and also the focus/active state.

Reply:
There are no types for that, so this is a workaround. Should I add the types?

Reply:
Well, we can use your solution for now. But it make sense to have these props available in Button/ButtonLink component as other devs can use them inside their projects.

Reply:
@vepor I agree but I would rather do that in a separate PR.